app/packages/utilities/src/size-bytes-estimate.ts

```diff
@@ -0,0 +1,68 @@
+const OBJECT_CLASS_CACHE = {};
+
+const objectClass = (object: object) => {
+  const prototype = Object.prototype.toString.call(object);
+  if (!OBJECT_CLASS_CACHE[prototype]) {
+    OBJECT_CLASS_CACHE[prototype] = prototype.slice(8, -1);
+  }
+  return OBJECT_CLASS_CACHE[prototype];
+};
+
+const isObject = (object: object) => {
+  const oClass = objectClass(object);
+  return oClass === "Object" || oClass === "Array";
+};
+
+type SizeTypes = null | object | string | undefined;
+
+type SizerTypes = SizeTypes | Array<SizeTypes>;
+
+const sizer = (object: SizerTypes) => {
+  if (object !== null && object !== undefined) {
```

Comment:
I am confused by this logic. Why are we returning 1 if the object is defined? Also I thought you could just 

```
if (object) {
 //do whatever
}
```

to determine if it's not null or undefined

Reply:
Woops! Thank you